dashboard_description_plugin/src/dp_dashboard_description_plugin/components/InlineAttribute.tsx

```diff
@@ -0,0 +1,60 @@
+// (C) 2022-2023 GoodData Corporation
+import React, { useMemo } from "react";
+import { UseCancelablePromiseStatus } from "@gooddata/sdk-ui";
+
+import { AttributeConfig, PluginMessages } from "../interface.js";
+import { TooManyDataPointsError } from "../utils.js";
+
+import { InlineValue } from "./InlineValue.js";
+
+interface IInlineAttributeProps {
+    status: UseCancelablePromiseStatus;
+    value: string | undefined;
+    messages: PluginMessages;
+    config: AttributeConfig;
+    error: Error | null | undefined;
+}
+
+/**
+ * Handles rendering of email or link values.
```

Comment:
@dremileta I would put this a bit differently. The component handles rendering of attributes, and if the attribute is an email or a link, it renders them as such.